motors_roboteq_canopen.orogen

```diff
@@ -69,7 +83,7 @@ task_context "Task", subclasses: "canopen_master::SlaveTask" do
     # The analog inputs
     output_port "analog_inputs", "/std/vector</raw_io/Analog>"
 
-    runtime_states "INPUT_TIMEOUT"
+    runtime_states :JOINT_CMD_TIMEOUT
```

Comment:
This is a non-backward-compatible change. I don't think it is warranted in this instance.